src/main/java/cat/udl/eps/softarch/demo/domain/Schedule.java

```diff
@@ -16,16 +16,17 @@ public class Schedule {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private int Id;
+    private long Id;
+
 
     @NotNull
-    private Time startTime;
+    private String startTime;

```

Comment:
The use of String is not recommended, it is better to use a specific Java Class. Moreover, as the schedule needs to have a start and end date in addition to time, the recommendation is to use java.time.ZonedDateTime. It is recommended to use datetimes with time zones to facilitate consuming it from different parts of the world, which should be expected in case of global APIs